src/main/java/com/stosik/parking/reservation/domain/Meter.java

```diff
@@ -0,0 +1,11 @@
+package com.stosik.parking.reservation.domain;
+
+import com.stosik.parking.reservation.domain.model.Reservation;
+import com.stosik.parking.reservation.dto.CreateReservationCommand;
+
+interface Meter
```

Comment:
co nam daje ten interfejs?

Reply:
Wykorzystalem go jedynie zeby wstzyknac interfejs Meter zamiast ParkingMeter i zachowac loose coupling na przyszlosc, aczkolwiek zastanawiajac sie w tym momencie to jest to spore naduzycie z mojej strony.